components/ILIAS/Forum/PRIVACY.md

```diff
@@ -0,0 +1,101 @@
+
+The **ID of the creating user** and the time of creation are saved for the entire forum. If the forum settings are changed, the **ID of the user making the change** and the time of the change are saved.
+
+The **title, description** and a manually designed **start page** of the forum may contain personal information provided by the creator.
+
+The **titles and texts of the posts** and topics in the forum are created by the forum participants themselves and may contain personal information, e.g. the names of other forum participants. Posts may also contain images and file attachments.
+
+With every post or draft of a post, the **ID** of the user who created it and the **time** of creation and publication are saved. When changes are made (including censorship or re-release), the time and ID of the user making the change are saved.
+
+If anonymization is set in the forum, the ID of the user and the **pseudonym** entered are saved internally with a post.
+
+The **moderator** function of a user is saved via the ILIAS access control system by the moderator role they belong to. For **censored posts**, the time and comment are saved and are linked to the ID of the censor.
+
+**Notification settings** for the forum or individual topics are saved for users (notification of new topics, modification or censorship of posts, deletion of posts or topics). The settings are linked to the **IDs** of the user who made them and to whom they apply.
+
+The **read status** of posts and the **sorting** of topics is saved for individual users and used to customize the display.
+
+If forum topics can be given a 5-star rating, **individual ratings** are saved by the rating component of ILIAS.
+
+## Data being presented
+
+The title, description, start page of the forum as well as the titles of the topics and the posts may contain **manually created personal information**.
+
+If a post was written without pseudonymization, the **ILIAS username** and the **date of creation** are visible to other users with read access to the forum. In the case of highlighted posts by moderators, a **Moderation** appears. In the case of subsequent changes, the user name and the date of the **change** are visible.
+
+If a user has published a personal profile, the **first name** and **surname** are also displayed. Depending on the profile settings, a **portrait** or **letter avatar** is displayed. If a forum is enabled for anonymous access via the access control system, this data is only displayed for anonymous users if the profile has been made publicly visible.
+
+If a post has been written with a **pseudonym**, the pseudonym entered appears instead of the username and is recognizable as such.
+
+If **statistics** are activated in the forum, a table is available for all users with read access, in which at least the **username** and the **number of posts** are listed for users with posts. If the personal profile is enabled, the first name and surname are also displayed. 
```

Comment:
Slightly change to:
"If **statistics** are activated in the forum, a table is available for all users with read access, listing at least the **username** and the **number of posts** for users with posts. If the personal profile is enabled, the **first name** and **surname** are also displayed."